schemas/common/email-interaction.schema.json

```diff
@@ -0,0 +1,136 @@
+  "definitions": {
+    "emailinteraction": {
+      "properties": {
+        "xdm:priority": {
+          "title": "Email's priority",
+          "type": "string",
+          "description": "Captures the priority set by the person who sent the email"
+        },
+        "xdm:sentDateTime": {
+          "title": "Date and time the email was sent",
+          "type": "string",
+          "format": "date-time",
+          "description": "Captures the date and time the email correspondence was sent to an individual or a group of people"
+        },
+        "xdm:receivedDateTime": {
+          "title": "Date and time the email was received",
+          "type": "string",
+          "format": "date-time",
+          "description": "Captures the date and time when the email was received"
+        },
+        "xdm:body": {
+          "title": "Body of the mail",
+          "type": "string",
+          "description": "The contents of the email after substitution of the placeholders in the template"
+        },
+        "xdm:attachment": {
+          "title": "Email has attachments",
+          "type": "boolean",
+          "description": "A flag represent whether any attachment is part of the email message"
+        },
+        "xdm:quarantine": {
+          "title": "quarantine message",
+          "type": "boolean",
+          "description": "A flag to represent whether the email message is a spam"
+        },
+        "xdm:sizeInBytes": {
+          "title": "Email message size",
+          "type": "number",
+          "description": "Record the size of the email message in bytes"
+        },
+        "xdm:failureReasonCode": {
+          "title": "Reason code for failure",
+          "type": "string",
+          "description": "Indicates the error code for not able to deliver the email message"
+        },
+        "xdm:failureReasonMessage": {
+          "title": "Reason Message for failure",
+          "type": "string",
+          "description": "Indicates the full message for the error code, for not able to deliver the message"
+        },
+        "xdm:fromAddress": {
+          "title": "Person sending the email communication",
+          "type": "string",
+          "format": "email",
+          "description": "Person who is sender of the email message"
+        },
+        "xdm:toAddress": {
+          "type": "array",
+          "title": "An array of email addresses for the To list",
+          "description": "Person or list of people who are in the To list of the email message",
+          "items": {
+            "type": "string",
+            "format": "email"
+          }
+        },
+        "xdm:ccAddress": {
+          "type": "array",
+          "title": "An array of email addresses for the cc list",
+          "description": "Person or list of people who are in the cc list of the email message",
+          "items": {
+            "type": "string",
+            "format": "email"
+          }
+        },
+        "xdm:bccAddress": {
```

Comment:
If this address is an email address, we should be using the email data type here https://ns.adobe.com/xdm/context/emailaddress